### ISSUE TITLE
`no-var-keyword` doesn't reject `export var` statements

### ISSUE THREAD

**Issue by giladgray** [palantir/tslint#1256]
### Bug Report

- __TSLint version__: `3.10.1`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: Node.js API & gulp-tslint

#### TypeScript code being linted

```ts
export var OverlayFactory = React.createFactory(Overlay);
```

with `tslint.json`:

```json
"no-var-keyword": true,
```

#### Actual behavior

Does not reject `var` keyword

#### Expected behavior

Would reject `var` keyword.

Removing the `export` keyword (`var name = ...`) does lint correctly.
